pkg/printers/template_flags.go

```diff
@@ -37,9 +39,9 @@ type GoTemplatePrintFlags struct {
 // ToPrinter receives an templateFormat and returns a printer capable of
 // handling --template format printing.
 // Returns false if the specified templateFormat does not match a template format.
-func (f *GoTemplatePrintFlags) ToPrinter(templateFormat string) (ResourcePrinter, error) {
+func (f *GoTemplatePrintFlags) ToPrinter(templateFormat string) (printers.ResourcePrinter, error) {
```

Comment:
This shouldn't be required.  Just keep the interfaces overlapping.

Reply:
Done